Changelog.md

```diff
@@ -1,8 +1,14 @@
-## Unreleased
+## 0.7.2
+
 ### Bug Fixes
 
+* Fix a bug in the `Applicative` and `Functor` instances of the `Fold`
+  data type.
+* Fix a bug that caused the streamly builds to fail on 32-bit machines.
```

Comment:
Looking at this section looks like there are many bug fixes. We can put the build related fixes in a separate "Build Issues" section.